src/main/java/de/teddy/advancementhunt/mysql/MySQLManager.java

```diff
@@ -38,7 +38,7 @@ public MySQLManager(ConfigManager configManager)
             ArrayList<Long> seeds = (ArrayList<Long>) AdvancementHunt.getInstance().getAdvancementSeed().getConfig().getLongList("World_Seeds");
             return seeds;
         }
-        this.table_name = configManager.getMessage("Game.MySQL.Table_Name");
+        this.table_name = configManager.getMessage("Game.MySQL.table_prefix") + "advancements";
```

Comment:
`advancements` isnt a suitable name for this table since it houses both Advancements and World Seeds. Change name to `storage` or add another table for world seeds, either is a suitable solution.

Reply:
alright